docs/developing/deploy_facilities/interacting_with_spl_tokens.md

```diff
@@ -4,18 +4,90 @@ proofedDate: na
 iterationBy: na
 includedInSite: true
 approvedBy: na
-comments: #1 TODO update text #fix broken link -- see inline comment
+comments:
+boilerPlatable: yes -- we could have an item providing demo
 ---
 
-The ERC-20 SPL wrapper contract provides access to native Solana tokens registered in the SPL token contract, through the ERC-20 interface.
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-This allows Solana applications to interact with EVM (Solidity, Vyper, etc.) bytecode contracts. The ERC-20 SPL wrapper can also be used to transfer funds in Solana tokens using Ethereum wallets such as MetaMask.
+The ERC-20 SPL wrapper contract provides access to native Solana tokens, i.e. those registered in the SPL token contract, through the ERC-20 interface. 
 
-The contract is implemented in Rust as part of the Neon EVM program.
+This allows Solana liquidity to be available to EVM (Solidity, Vyper, etc.) bytecode contracts, i.e. this ERC-20 SPL wrapper allows Ethereum wallets such as MetaMask to transact with SPL tokens.
+
+:::info
+To be able to use an SPL token from a Solana account balance, it must be transferred to a NeonEVM account via the [NeonPass SDK](/docs/developing/integrate/neon_transfer_sdk).
+:::
+
+Two contracts are available on Neon EVM that enable SPL tokens to be transacted as ERC-20 compliant. It's vital that you understand which to use:
```

Comment:
> Two contracts are available on Neon EVM

Incorrect wording. Does not make sense

Reply:
OK key to feedback is that it is precise and actionable! let's Huddle and make that so...